docs/articles_en/documentation/openvino-extensibility/openvino-plugin-library/transformation-python-api/matcher-pass.rst

```diff
@@ -0,0 +1,46 @@
+.. {#openvino_docs_Extensibility_UG_matcher_pass}
+
+OpenVINO Model Pass Python API
```

Comment:
Why do we need a separate page for Python API
I believe it's better to update C++ page with these python examples